src/index.js

```diff
@@ -161,11 +161,7 @@ export class Step extends Component {
 
     return (
       <Consumer>
-        {({ currentStep, ...childProps }) => (
-          <div hidden={currentStep !== id}>
-            {this._renderChildren(childProps)}
-          </div>
-        )}
+        {({ currentStep, ...childProps }) => this._renderChildren(childProps)}
```

Comment:
uhm now currentStep isn't being handled at all. make sure do do a conditional render for currentStep